gradle/lib/dependabot/gradle/file_fetcher.rb

```diff
@@ -82,24 +87,28 @@ def subproject_buildfiles(root_dir)
           .subproject_paths
 
         subproject_paths.filter_map do |path|
+          @buildfile_name = T.let(@buildfile_name, T.nilable(String))
           if @buildfile_name
-            fetch_file_from_host(File.join(root_dir, path, @buildfile_name))
+            buildfile_path = T.let(File.join(root_dir, path, @buildfile_name), String)
+            fetch_file_from_host(buildfile_path)
           else
             buildfile(File.join(root_dir, path))
           end
-        rescue Dependabot::DependencyFileNotFound
-          # Gradle itself doesn't worry about missing subprojects, so we don't
-          nil
+      rescue Dependabot::DependencyFileNotFound
+        # Gradle itself doesn't worry about missing subprojects, so we don't
+        nil
```

Comment:
It looks like the indentation was decreased a little here.

Reply:
@raj-meka Please run it through **rubocop -A** to see if this autocorrects.  whitespace doesn't seem like a big deal, but that rule should just check for issues which have already been corrected and make the code less likely to have issues when someone else rebases.

Reply:
@raj-meka it doesn't look like you resolved this